app/view/field/YearField.js

```diff
@@ -0,0 +1,16 @@
+Ext.define('CpsiMapview.view.field.YearField', {
+    extend: 'Ext.form.field.Number',
+    xtype: 'cmv_yearfield',
+    fieldStyle: 'text-align: right;', // TODO has no effect?
```

Comment:
Can you check if this is still a valid ExtJS property? If not then it should be removed

Reply:
It is!